conductor-client/src/main/kotlin/com/openlattice/organizations/OrganizationMetadataEntitySetsService.kt

```diff
@@ -254,25 +284,29 @@ class OrganizationMetadataEntitySetsService(
 
             val tableColumnEntities = columnsByTableId.getOrDefault(table.id, listOf()).associate { column ->
                 columnEntityKeyIds.getValue(AclKey(table.id, column.id)) to mutableMapOf<UUID, Set<Any>>(
-                        propertyTypes.getValue(ID).id to setOf(column.id.toString()),
-                        propertyTypes.getValue(DATASET_NAME).id to setOf(table.name),
-                        propertyTypes.getValue(COL_NAME).id to setOf(column.name),
-                        propertyTypes.getValue(ORG_ID).id to setOf(organizationId.toString()),
-                        propertyTypes.getValue(TYPE).id to setOf(column.dataType.toString()),
-                        propertyTypes.getValue(DESCRIPTION).id to setOf(column.description)
+                    propertyTypes.getValue(COL_NAME).id to setOf(column.name),
+                    propertyTypes.getValue(DATASET_ID).id to setOf(table.id),
+                    propertyTypes.getValue(DATASET_NAME).id to setOf(table.name),
+                    propertyTypes.getValue(DATA_TYPE).id to setOf(column.dataType.toString()),
+                    propertyTypes.getValue(DESCRIPTION).id to setOf(column.description),
+                    propertyTypes.getValue(ID).id to setOf(column.id.toString()),
+                    propertyTypes.getValue(INDEX).id to setOf(column.ordinalPosition),
+                    propertyTypes.getValue(ORG_ID).id to setOf(organizationId),
+                    propertyTypes.getValue(TITLE).id to setOf(column.title)
                 )
             }
 
             columnEntities.putAll(tableColumnEntities)
 
             datasetEntities[datasetEntityKeyIds.getValue(table.id)] = mutableMapOf(
-                    propertyTypes.getValue(PGOID).id to setOf(table.oid),
-                    propertyTypes.getValue(ID).id to setOf(table.id.toString()),
-                    propertyTypes.getValue(DATASET_NAME).id to setOf(table.name),
-                    propertyTypes.getValue(STANDARDIZED).id to setOf(false),
-                    propertyTypes.getValue(COL_INFO).id to setOf(mapper.writeValueAsString(tableColumnEntities.values))
+                propertyTypes.getValue(DATASET_NAME).id to setOf(table.name),
+                propertyTypes.getValue(DESCRIPTION).id to setOf(table.description),
+                propertyTypes.getValue(ID).id to setOf(table.id.toString()),
+                propertyTypes.getValue(ORG_ID).id to setOf(organizationId),
+                propertyTypes.getValue(PG_OID).id to setOf(table.oid),
+                propertyTypes.getValue(STANDARDIZED).id to setOf(false),
+                propertyTypes.getValue(TITLE).id to setOf(table.title)
```

Comment:
we should probably add a property type + store schema here too

Reply:
I prefer to do that in a follow-up pull request.